libraries/3rdparty/mbedtls/include/mbedtls/mbedtls_error.h

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Also does this custom file belong in the third party folder?

Reply:
Have relocated files to `mbedtls_utils`